content/help/enter-site-details.md

```diff
@@ -0,0 +1,23 @@
+# Help Page: Entering Site Details for Solar Energy Forecasting
+
+Once you've added a site to our solar energy forecasting web app, the next step is to enter the site details. This information is crucial for generating accurate and relevant forecast data. Here are the fields we need you to fill out:
+
+## Site Name
+
+This field is for you to provide a specific name to identify your site. You can choose any name that will help you easily recognize your site in the future.
+
+## Orientation
+
+The orientation field is where you enter the direction that your solar panels are pointing. The direction is measured in degrees, with 0 degrees being North and 180 degrees being South. If you're unsure of the orientation, you may be able to find this information in the manuals provided by the installer. Alternatively, you can use a compass app with your back to the solar panels to determine the orientation.
+
+## Tilt
+
+The tilt field refers to the angle at which your solar panels are positioned. This is also sometimes called azimuth. You'll need to input the tilt in degrees. If you're unsure of the tilt, you may be able to find this information in the manuals provided by the installer.
```

Comment:
change `The tilt field refers to the angle at which your solar panels are positioned. This is also sometimes called azimuth.` to `The tilt field refers to the angle (azimuth) at which your solar panels are positioned.`